src/app/store.ts

```diff
@@ -1,16 +1,23 @@
 import { configureStore } from '@reduxjs/toolkit';
-import count from '../features/count/countSlice';
+import type { Meta } from '../common/types';
 import auth from '../features/auth/authSlice';
-import layout from '../features/layout/layoutSlice';
+import count from '../features/count/countSlice';
 import icons from '../common/slices/iconSlice';
+import layout from '../features/layout/layoutSlice';
+import profile from '../features/profile/profileSlice';
 
-const createStore = () =>
-  configureStore({
-    reducer: { count, auth, layout, icons },
-  });
+const createStore = (preloadedState?: Meta) => {
+  const config = {
+    devTools: true,
+    preloadedState: preloadedState ? preloadedState : {},
```

Comment:
Maybe it would be neater if the `preloadedState` var was moved completely out of `createStore` and having `createTestStore` be the function that could handle preloaded state

Reply:
This change is included in the latest commit.